lib/mysql.php

```diff
@@ -61,9 +61,15 @@ function mysql_connect(
                 return \Dshafik\MySQL::$connections[$hash]['conn'];
             }
 
+            if (strpos($hostname, ':') !== false) {
```

Comment:
Should be `strpos($hostname, ':', 2)` since persistent connections will begin with the string "p:" prepended.